dts/bindings/dac/dac-controller.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2020 Libre Solar Technologies GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+# Common fields for DAC controllers
+
+include: base.yaml
+
+properties:
+    label:
+      required: true
+
+    ref-internal:
+      type: int
+      required: true
+      description: Internal reference voltage in mV or 0 if not existing
```

Comment:
Instead of `0 if not existing` I suggest changing it to `required: false`.

Reply:
The internal reference is currently also exposed in the API. So if it's not set here, what should the API assume?
I think it makes sense to require defining it in the DTS. At least in the board-level DTS you should know if the Vref pin is wired to VCC or an external or internal reference.
In the Linux DTS they have a reference obtained from DTS in some way. But I'm not sure if that's possible in Zephyr aswell.

Reply:
How do you propose handling a DAC without any internal reference with the proposed API and DTS binding?

Reply:
Agree that didn't make much sense to specify a fixed value in the DTS, so I removed ref-internal setting from DTS again. Actual value of the reference should be handled by the application.